docs/service-catalog/docs/03-04-experimental-features.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Experimental features
+type: Details
+---
+
+Currently Service Catalog requires its own instance of api-server and etcd. This adds additional complexity to the cluster configuration and increases
+maintenance costs. Kyma developers are contributing to the Service Catalog project to remove dependency to these external components and replace them 
+with native K8S solution - Custom Resource Definition.
+
+### Enable CRDs
+
+To enable the CRDs feature you have to override parameters `service-catalog-apiserver.enabled` and `service-catalog-crds.enabled`
+in the installer-config file. Modify or add the `service-catalog-overrides` config map:  
```

Comment:
Mention exactly which file to modify(link to local config) and in which one to add a CM (link to cluster cluster)